adopters.geojson

```diff
@@ -31,6 +31,7 @@
     {"type": "Feature", "properties": {"institution": "Technical University of Applied Sciences Lübeck"}, "geometry": {"type": "Point", "coordinates": [10.7002646, 53.837048]}},
     {"type": "Feature", "properties": {"institution": "Hochschule Bochum, Campus Velbert/Heiligenhaus"}, "geometry": {"type": "Point", "coordinates": [6.96794, 51.32762]}},
     {"type": "Feature", "properties": {"institution": "Hochschule Bochum"}, "geometry": {"type": "Point", "coordinates": [7.27248, 51.44711]}},
+    {"type": "Feature", "properties": {"institution": "INSA Lyon"}, "geometry": {"type": "Point", "coordinates": [45.7830905,4.8759893530497695]}},
```

Comment:
Thanks for adding your BBB! A caveat of this map is that the coordinates are flipped. Could you just change that?

```suggestion
    {"type": "Feature", "properties": {"institution": "INSA Lyon"}, "geometry": {"type": "Point", "coordinates": [4.8759893530497695, 45.7830905]}},
```

Reply:
Sorry, yes I definitely put the coordinates in the wrong order, compared to the other points !